rma/models/rma.py

```diff
@@ -980,6 +980,13 @@ def _create_receptions_from_product(self):
             ),
             view="stock.view_picking_form",
         )
+        if self.operation_id.name == "Replace":
+            self._action_launch_stock_rule(
```

Comment:
@Hardik-OSI I wouldn't have based my comparison on record name. Moreover as the record can be deleted (noupdate=1).

Adding an option on rma.operation that triggers the stock rule is better IMHO.